docs/userguides/projects.md

```diff
@@ -46,19 +46,22 @@ For more information on compiling your project, see [this guide](./compile.html)
 After compiling, the contract containers are accessible from the `project` manager.
 Deploy them in the `console` or in scripts; for example:
 
-```python
-from ape import accounts, project
+```{eval-rst}
+
+.. doctest::
 
-account = accounts.load("my_account_alias")
-account.deploy(project.MyContract)
+>>> from ape import accounts, project, networks
+>>> with networks.parse_network_choice("ethereum:local:test") as provider:
+...     account = accounts.load("my_account_alias")
+...     account.deploy(project.MyContract)
 ```
 
 **NOTE**: You can also deploy contracts from the container itself:
 
 ```python
 from ape import accounts, project
 
-account = accounts.load("my_account_alias")
+account = accounts.load(f"{my_account_alias}")
```

Comment:
this will likely always fail in a doc-test i think that is ok. is there a way to ignore all ApeExceptions?

Reply:
Yes currently this is failing but if `my_account_alias = accounts.test_accounts[0].alias` works in the testsetup then I can try and use that instead. I'll look into ignoring ApeExceptions since that would be also useful in a few other userguides.